pyproject.toml

```diff
@@ -1,3 +1,39 @@
+[tool.mypy]
+strict_optional = true
+warn_return_any = true
+warn_no_return = true
+# TODO warn_unused_ignores = true
+warn_redundant_casts = true
+warn_incomplete_stub = true
+# TODO disallow_untyped_calls = true
+check_untyped_defs = true
+disallow_any_generics = true
+no_implicit_optional = true
+# TODO disallow_incomplete_defs = true
+# TODO disallow_subclassing_any = true
+disallow_untyped_decorators = true
+warn_unused_configs = true
+show_error_codes = true
+show_column_numbers = true
+ignore_missing_imports = true
+exclude = [
+    '\.git',
```

Comment:
We can use vscode's format document to standardize format on this file

Reply:
Formatted and moved pylint. Thanks!